autoemulate/emulators/__init__.py

```diff
@@ -12,13 +12,14 @@
 # REGISTRY keys are the class names (i.e. type(model).__name__)
 MODEL_REGISTRY = {
     "SecondOrderPolynomial": SecondOrderPolynomial(),
-    "RBF": RBF(),
+    "RadialBasisFunctions": RBF(),
     "RandomForest": RandomForest(),
     "GradientBoosting": GradientBoosting(),
-    "GaussianProcessSk": GaussianProcessSk(),
+    "GaussianProcesses": GaussianProcessSk(),
     "SupportVectorMachines": SupportVectorMachines(),
     "XGBoost": XGBoost(),
+    "NeuralNet": NeuralNetTorch(module="mlp"),
```

Comment:
Should we have a prefix of `NN` followed by `<module name>` for all `NeuralNetTorch` model? 

e.g.
- `"NNMLP": NeuralNetTorch(module="mlp")`
- `"NNRBF": NeuralNetTorch(module="rbf")`

so that
- users can immediately tell they belongs to `NeuralNetTorch`
- they will be close to each other when we sort all the model names alphabetically for printing. 
- in `pytest`, we can test every model that has prefix `NN` instead of hard-coding their names in our test cases.

Reply:
I like this suggestion - what do you think @mastoffel?

Reply:
both good suggestions! I'm on it.